lib/Conversion/TritonGPUToLLVM/PtxAsmFormat.cpp

```diff
@@ -74,6 +76,25 @@ SmallVector<PTXBuilder::Operand *, 4> PTXBuilder::getAllArgs() const {
   return res;
 }
 
+mlir::Value PTXBuilder::launch(ConversionPatternRewriter &rewriter,
+                               Location loc, Type resTy, bool hasSideEffect,
+                               bool isAlignStack,
+                               ArrayRef<Attribute> attrs) const {
+  auto *ctx = rewriter.getContext();
+  auto inlineAsm = rewriter.create<LLVM::InlineAsmOp>(
+      loc, resTy, getAllMLIRArgs(), // operands
+      dump(),                       // asm_string
+      getConstraints(),             // constraints
+      true,                         // has_side_effects
+      false,                        // is_align_stack
```

Comment:
Or here the hard-coded `false` should be replaced with `isAlignStack`?

Reply:
Yeah, I've corrected the code.